packages/visx-brush/src/BaseBrush.tsx

```diff
@@ -43,6 +43,7 @@ export type BaseBrushProps = {
   disableDraggingSelection: boolean;
   resetOnEnd?: boolean;
   useWindowMoveEvents?: boolean;
+  renderBrushHandles?: (props: BrushHandleRenderProps) => JSX.Element | null;
```

Comment:
Since this only renders one handle, I wonder if it makes sense to remove the `s` throughout? also should be able to use `ReactNode` for the return type which includes null.

```suggestion
  renderBrushHandle?: (props: BrushHandleRenderProps) => React.ReactNode;
```